src/data.ts

```diff
@@ -68,7 +68,7 @@ export const initMetadataSubmission = (): MetadataSubmission => ({
   addressForm: initAddressForm(),
   contextForm: initContextForm(),
   multiOmicsForm: initMultiOmicsForm(),
-  packageName: "",
+  packageName: "", // TODO: CHANGE DEFAULT WHEN BACKEND IS UPDATED
```

Comment:
I think this alternative would be more instructive to other maintainers, but I am not sure it accurately represents the original author's intent.

```suggestion
  packageName: "", // TODO: CHANGE DEFAULT VALUE TO EMPTY ARRAY WHEN BACKEND IS UPDATED
```